pkgs/by-name/go/gopeed/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  autoPatchelfHook,
+  dpkg,
+  makeWrapper,
+  wrapGAppsHook3,
+  libayatana-appindicator,
+  libayatana-indicator,
+  libdbusmenu,
+  ayatana-ido,
+  zenity,
+}:
+stdenv.mkDerivation rec {
+  pname = "gopeed";
+  version = "1.6.4";
+
+  src = fetchurl {
+    url = "https://github.com/GopeedLab/gopeed/releases/download/v${version}/Gopeed-v${version}-linux-amd64.deb";
+    hash = "sha256-tfBeoUlZOMmGeeKNgm22X/QzYKOUozI8dt3H76TiqB4=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    autoPatchelfHook
+    wrapGAppsHook3
+    makeWrapper
+  ];
+
+  buildInputs = [
+    libayatana-appindicator
+    libayatana-indicator
+    libdbusmenu
+    ayatana-ido
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+    cp -r opt $out/app
+    cp -r usr/share $out/share
+
+    runHook postInstall
+  '';
+
+  preFixup = ''
+    makeWrapper $out/app/gopeed/gopeed $out/bin/gopeed \
```

Comment:
This is in combination with wrapGAppsHook3 causing double wrapping. Please combine those wrappers into one.